platform/tyan-s1706/boot/config.h

```diff
@@ -0,0 +1,3 @@
+#include "boot-config.auto.h"
+#define CONFIG_DRAM_528
+#define CONFIG_DRAM_UART_57600
```

Comment:
We should consider making this platform 115200 baud if we have no reason not to. The Leopard uses 57600 for everything so that's why it used that baud.